base-server.d.ts

```diff
@@ -0,0 +1,704 @@
+import { Log, Store, ServerConnection, TestTime } from '@logux/core'
+import { Server as HTTPServer } from 'http'
+
+import { Context, ChannelContext, LoguxPatternParams } from './context'
+import { ServerClient } from './server-client'
+
+/**
+ * Logux meta
+ */
+export type LoguxMeta = {
+  id: string
+  time: number
+  subprotocol: LoguxBaseServerOptions['subprotocol']
+  server: string
+  reasons: string[]
+  channels?: string[]
+  clients?: string[]
+  status?: 'waiting' | 'add' | 'clean' | 'processed' | 'subscribed'
```

Comment:
Note that in Logux Core I hav comments for meta fields
https://github.com/logux/client/blob/master/client/index.d.ts#L22

You can copy them from https://logux.io/guide/concepts/meta/#server-meta-keys